esp/src/src-react/hooks/grid.tsx

```diff
@@ -25,7 +28,7 @@ export function useGrid({ store, query = {}, sort = [], columns, getSelected, fi
     const [grid, setGrid] = React.useState<any>(undefined);
     const [selection, setSelection] = React.useState([]);
 
-    const Grid = React.useMemo(() => () => <DojoGrid
+    const Grid = React.useCallback(() => <DojoGrid
```

Comment:
Why switch to useCallback?

Reply:
Just for consistency as it is really a callback.